Apps/WebClient/src/ClientApp/test/error.test.ts

```diff
@@ -0,0 +1,36 @@
+import { mount, shallowMount } from '@vue/test-utils'
+import NotFoundView from '@/views/errors/notFound.vue'
+import UnauthorizedView from '@/views/errors/unauthorized.vue'
+import ErrorComponent from '@/components/error.vue'
+import PageError from '@/models/pageError'
+
+describe('ErrorComponent', () => {
+    test('renders properties correctly', () => {
+        let error: PageError = new PageError('777', 'ERROR_NAME', 'ERROR_MESSAGE')
+        const wrapper = shallowMount(ErrorComponent, { propsData: { error } });
+
+        expect(wrapper.find("h1").text()).toBe(error.code)
+        expect(wrapper.find("h2").text()).toBe(error.name)
+        expect(wrapper.find("p").text()).toBe(error.message)
+    })
+})
+
+describe('NotFoundView', () => {
+    test('renders properties correctly', () => {
+        const mountWrapper = mount(NotFoundView);
```

Comment:
it doesn't matter for this smaller tests but ideally the wrapper should be declared/initiated outside this test but inside the describe, so we can reuse it through more than 1 test, you could even break each expect into one specific test, eg: 

describe('ErrorComponent', () => {
  let error: PageError = new PageError('777', 'ERROR_NAME', 'ERROR_MESSAGE')
  const wrapper = shallowMount(ErrorComponent, { propsData: { error } });

  test('renders header 1 correctly', () => {
    expect(wrapper.find("h1").text()).toBe(error.code)
  })

  test('renders header 2 correctly', () => {
    expect(wrapper.find("h2").text()).toBe(error.name)
  })

  test('renders paragraph correctly', () => {
    expect(wrapper.find("p").text()).toBe(error.name)
  })
})

Reply:
That looks way cleaner! Do you know if any state declared on the described will be cleaned after each test?